content/id/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,234 @@
+| `node.kubernetes.io/unreachable`         | NoExecute  | 1.13+   | Pod DaemonSet tidak akan menjadi _evicted_ ketika ada masalah node seperti partisi jaringan.                 |
+| `node.kubernetes.io/disk-pressure`       | NoSchedule | 1.8+    |                                                                                                              |
+| `node.kubernetes.io/memory-pressure`     | NoSchedule | 1.8+    |                                                                                                              |
+| `node.kubernetes.io/unschedulable`       | NoSchedule | 1.12+   | Pod DaemonSet mentoleransi atribut `unschedulable` _default scheduler_.                                      |
+| `node.kubernetes.io/network-unavailable` | NoSchedule | 1.12+   | Pod DaemonSet yang menggunakan jaringan host mentoleransi atribut `network-unavailable` _default scheduler_. |
+
+
+
+## Berkomunikasi dengan Pod Daemon
+
+Beberapa pola yang mungkin digunakan untuk berkomunikasi dengan Pod dalam DaemonSet, yaitu:
+
+- **Push**: Pod dalam DaemonSet diatur untuk mengirim pembaruan status ke servis lain,
+  contohnya _stats database_. Pod ini tidak memiliki klien.
+- **NodeIP and Known Port**: Pod dalam DaemonSet dapat menggunakan `hostPort`, sehingga Pod dapat diakses menggunakan IP node. Klien tahu daftar IP node dengan suatu cara, dan tahu port berdasarkan konvensi.
+- **DNS**: Buat [headless service](/docs/concepts/services-networking/service/#headless-services) dengan pod selektor yang sama,
+  dan temukan DaemonSet menggunakan _resource_ `endpoints` atau mengambil beberapa A _record_ dari DNS.
+- **Service**: Buat Servis dengan pod selektor yang sama, dan gunakan Servis untuk mengakses daemon pada
+  node random. (Tidak ada cara mengakses spesifik node)
+
+## Melakukan Pembaruan DaemonSet
+
+Jika label node berubah, DaemonSet akan menambahkan Pod ke node cocok yang baru dan menghapus Pod dari
+node tidak cocok yang baru.
+
+Anda bisa mengubah Pod yang dibuat DaemonSet. Namun, Pod tidak membolehkan perubahan semua _field_.
+Perlu diingat, _controller_ DaemonSet akan menggunakan templat yang asli di waktu selanjutnya 
+node baru (bahkan dengan nama yang sama) dibuat.
+
+Anda bisa menghapus DaemonSet. Jika Anda spesifikasikan `--cascade=false` dengan `kubectl`, maka
+Pod akan dibiarkan pada node. Jika Anda pada waktu kemudian membuat DaemonSet baru dengan selektor
+yang sama, DaemonSet yang baru akan mengadopsi Pod yang sudah ada. Jika ada Pod yang perlu diganti,
+DaemonSet akan mengganti sesuai dengan `updateStrategy`.
+
+Anda bisa [melakukan rolling update](/docs/tasks/manage-daemon/update-daemon-set/) pada DaemonSet.
+
+## Alternatif DaemonSet
+
+### _Init Scripts_
+
+Anda mungkin menjalankan proses _daemon_ dengan cara menjalankan mereka langsung pada node (e.g. 
```

Comment:
```suggestion
Kamu mungkin menjalankan proses _daemon_ dengan cara menjalankan mereka langsung pada node (e.g. 
```

Reply:
Also we need to make _daemon_ consistent (some places have daemon with italic, some without)

Reply:
I'll go with all italicized.